src/base-apis.js

```diff
@@ -1683,6 +1683,25 @@ MatrixBaseApis.prototype.getThirdpartyLocation = function(protocol, params) {
     );
 };
 
+/**
+ * Get information on how a specific user on a third party protocol
+ * may be reached.
+ * @param {string} protocol The protocol given in getThirdpartyProtocols()
+ * @param {object} params Protocol-specific parameters, as given in the
+ *                        response to getThirdpartyProtocols()
+ * @return {module:client.Promise} Resolves to the result object
+ */
+MatrixBaseApis.prototype.getThirdpartyUser = function(protocol, params) {
+    const path = utils.encodeUri("/thirdparty/user/$protocol", {
+        $protocol: protocol,
+    });
+
+    return this._http.authedRequestWithPrefix(
+        undefined, "GET", path, params, undefined,
+        httpApi.PREFIX_UNSTABLE,
```

Comment:
This is in the v2 API now so if we're adding it, we should probably add it using the stable endpoint.

Reply:
Should the same be tweaked for `getThirdpartyProtocols` ? I assume they go together.

Reply:
Good question - since these are existing APIs, we should probably leave them using `unstable` for a while as not all servers will have updated to expose the APIs on the stable path yet.